src/components/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import classNames from 'classnames';
+import { Link, useLocation } from 'react-router-dom';
+import { Person } from '../types';
+import React, { FC } from 'react';
+type Props = {
+  person?: Person;
+};
+export const PersonLink: FC<Props> = ({ person }) => {
+  const location = useLocation();
+  const searchParams = location.search;
+
+  return (
+    <Link
+      to={`/people/${person?.slug}${searchParams}`}
+      className={classNames({
+        'has-text-danger': person?.sex === 'f',
```

Comment:
Use enum here for 'f' 
<img width="427" alt="image" src="https://github.com/user-attachments/assets/8ccf9af4-3d4f-43fc-9150-35ce75213458">